.gitignore

```diff
@@ -25,6 +25,7 @@ dist/
 src/renderer/angular/app.templates.js
 static/stylesheets/css/main.css
 static/stylesheets/css/*
+storybook-static/
```

Comment:
Is this windows specific?

Reply:
That folder appeared when I ran the build storybook command and I didn't want to add generated files to the repository.

Reply:
Ok. must be a windows specific thing for storybook.